models/lobby/lobby.go

```diff
@@ -148,6 +148,13 @@ func (lobby *Lobby) AddPlayer(player *models.Player, slot int) error {
 	return nil
 }
 
+func (lobby *Lobby) RemovePlayer(player *models.Player) {
+	slot, _ := lobby.GetPlayerSlot(player)
+	//We'll be not checking for errors, since the socket handler has already
```

Comment:
you should still check for an error - an if check is much cheaper than a potential program crash when someone else uses this method and doesn't check the input before